pkgs/development/tools/nsis/default.nix

```diff
@@ -28,30 +29,34 @@ stdenv.mkDerivation rec {
   '';
 
   nativeBuildInputs = [ sconsPackages.scons_3_1_2 ];
-  buildInputs = [ zlib ];
+  buildInputs = [ zlib libiconv ];
+
+  preBuild = lib.optionalString stdenv.isLinux ''
+    sconsFlagsArray+=("PATH=$PATH")
+  '' + ''
+    mkdir -p $out/tmp/include
+    cp ${zlib.dev}/include/* $out/tmp/include
+    cp ${libiconv}/include/* $out/tmp/include
+    sconsFlagsArray+=("APPEND_CPPPATH=$out/tmp/include")
```

Comment:
Do we really want to keep this in the output?

Reply:
Nope, used `symlinkJoin` instead.